testutil-server/src/main/java/io/spine/testing/server/projection/ProjectionEventDispatcher.java

```diff
@@ -95,9 +98,8 @@ private TestProjectionEndpoint(EventEnvelope event) {
     private static <I, P extends Projection<I, S, ?>, S extends Message>
     ProjectionRepository<I, P, S> mockRepository() {
         TestProjectionRepository mockedRepo = mock(TestProjectionRepository.class);
-        TestProjectionRepository.TestLifecycle mockedLifecycle =
-                mock(TestProjectionRepository.TestLifecycle.class);
-        when(mockedRepo.lifecycleOf(any())).thenReturn(mockedLifecycle);
+        when(mockedRepo.lifecycleOf(any())).thenCallRealMethod();
+        when(mockedRepo.getEntityStateType()).thenReturn(TypeUrl.of(Any.class));
```

Comment:
Can't we ask for an `Class<S> entityStateClass` in parameters and return a real value here?